docs/docs/unit-testing.md

```diff
@@ -229,16 +229,13 @@ Then update the configuration in `jest.config.js`, like so:
 ```js:title=jest.config.js
 module.exports = {
   transform: {
-    "^.+\\.tsx?$": "ts-jest",
-    "^.+\\.jsx?$": "<rootDir>/jest-preprocess.js",
+    "^.+\\.[jt]sx?$": "<rootDir>/jest-preprocess.js",
   },
-  testRegex: "(/__tests__/.*|(\\.|/)(test|spec))\\.([tj]sx?)$",
   moduleNameMapper: {
     ".+\\.(css|styl|less|sass|scss)$": "identity-obj-proxy",
     ".+\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
       "<rootDir>/__mocks__/file-mock.js",
   },
-  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
```

Comment:
I see that `jest` have our current/previous `testRegex`/`moduleFileExtensions` settings as default now:

https://jestjs.io/docs/en/configuration.html#modulefileextensions-array-string
> moduleFileExtensions [array]
> Default: ["js", "json", "jsx", "ts", "tsx", "node"]

https://jestjs.io/docs/en/configuration.html#testregex-string-array-string
> testRegex [string | array]
> Default: (/__tests__/.*|(\\.|/)(test|spec))\\.[jt]sx?$

Below section you added about updating `jest.preprocess.js` there are 2 paragraphs explaining those setting fields which now seem redundant, so maybe we should remove them?

Reply:
Good spot. I've removed those two paragraphs and pushed the changes.